server/app/services/geo/AddressLocation.java

```diff
@@ -0,0 +1,42 @@
+package services.geo;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class AddressLocation {
```

Comment:
suggestion (non-blocking): maybe `AddressCoordinates` would be more specific a name?